addons/mail/static/src/core/web/command_category.js

```diff
@@ -0,0 +1,9 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+const commandCategoryRegistry = registry.category("command_categories");
+
+commandCategoryRegistry
+    .add("discuss_mentioned_threads", { namespace: "@", name: "Mentioned" }, { sequence: 10 })
+    .add("discuss_recent_threads", { namespace: "#", name: "Recent" }, { sequence: 10 });
```

Comment:
`Mentioned` should have precedence over `Recent`, so theoretically the former should strictly have lower sequence and the latter. It's in the right order because of implementation details that first registered come first if same sequence, but it's weak to rely on that.

Reply:
They are different, one triggered with @ and one triggered with #, in different categories

Reply:
I was confused by name in registry: both have `threads` in their name, so I thought they represent different categories that might be displayed at the same time.
I would just remove `_threads` in suffix, it doesn't add much value and makes it harder to read

Reply:
I understand you. This is because we have threads now even if when do a search in the partners category. This is a little bit confusing but the spec decides the `Mentioned` should be displayed in `@` category.